framework/test/jest/unit/specs/controller/channels/base_channel.spec.js

```diff
@@ -156,6 +156,13 @@ describe('Base Channel', () => {
 		});
 	});
 
+	describe('#invokePublic', () => {
+		it('should throw TypeError', () => {
+			// Assert
+			expect(baseChannel.invokePublic()).rejects.toBeInstanceOf(TypeError);
```

Comment:
Don't forget to `return` or `await` expect statement when fulfilling promises. Otherwise it can give a false positive result by giving an unhandled promise rejection warning.

See: https://jestjs.io/docs/en/asynchronous#resolves-rejects